orttraining/orttraining/test/graph/optimizer_graph_builder_test.cc

```diff
@@ -374,7 +374,46 @@ TEST_F(OptimizerGraphBuilderTest, Allreduce_WithGradientAccumulation_WithMixedPr
   config.loss_scale_input_name = k_loss_scaling_factor_name;
   TestAllreduceOptimizerGraphBuilder(config, graph_);
 }
+TEST_F(OptimizerGraphBuilderTest, Adasum_Horovod_NoGradientAccumulation_NoMixedPrecision) {
```

Comment:
are these new UTs relevant to concat?

Reply:
they are not, had to add to get it to build locally. removing.